### DISCUSSION TITLE
Meal plan regressions from v0.5

### DISCUSSION THREAD

**Discussion by terrapinlogo** [mealie-recipes/mealie#1227] [General]
Just some features that don't appear to have made it to 1.0 that would be nice to have back.

Auto populate next n days/weeks/months with random recipes (following the new 1.0 rules)
  - option to only populate breakfast, lunch or dinner or all would be a nice addition.
 
![image](https://user-images.githubusercontent.com/5179945/168414964-f7c42fd0-d03a-42d2-ae55-0e79708673de.png)

Add all from selected meal plan to shopping list/ create new list

![image](https://user-images.githubusercontent.com/5179945/168415068-4f1311bb-1c24-44b2-bfa6-548f7506cd14.png)

![image](https://user-images.githubusercontent.com/5179945/168415053-e76ec993-f98f-4f8c-bca4-93dcd68513f4.png)

**Comment by FeralFlora** [comment]
This is sorely missed, because this random meal planning feature for the whole week was one of the main reasons I chose Mealie over the other options available.

**Comment by Blackclaws** [comment]
Tandoor Recipes even added this is in one of their latest releases, why was this removed exactly?

**Comment by boc-the-git** [comment]
@FeralFlora @Blackclaws, there is currently buttons to choose a random meal per day (see screenshot). Is the concern here ultimately something like having removed an option to perform what this button does, but for every day in the view?

![image](https://github.com/mealie-recipes/mealie/assets/3479092/b30c1705-d908-402b-8260-27d1e445d6ee)

**Reply by FeralFlora** [reply]
Yes, that's the concern. As @terrapinlogo's screenshots show, there used to be buttons so you could randomize the meal plan of a certain meal type (or even all meal types) for the whole week. Now, you have to do each day, one by one.

The same goes for the shopping list. Previously, you could add all recipes to the shopping list in one go. Now, you have to add each one individually.

**Reply by boc-the-git** [reply]
I think there's definitely a good argument something like this could be implemented, but I don't have good thoughts on how it could work from a UI/UX perspective.. 🤔
